### PR TITLE
Fixes #7826 & #7809 - blood decals being unanchored

### LINKED ISSUE #7826: Terror Spiders can wrap blood splatters.
**Problem Description**:
As a terror spider I was wrapping dropped weapons from victims, and I found I could wrap blood splatters on the ground. Muh Immulsions.

**What did you expect to happen**: 
For them to not be able to do that? Do they scoop it up with their little spider hands and wrap that? 

**What happened instead**:
Wrapping of a liquid. And a liquid splattered on the ground no less.

**Why is this bad/What are the consequences:**
I mean. I guess if you want a clean nest you can wrap the blood of your victims but it makes no sense.

**Steps to reproduce the problem**:
Capture person. Apply murder. Wrap blood.

**Possibly related stuff (which gamemode was it? What were you doing at the time? Was
anything else out of the ordinary happening?)**: Nope.

I think this might also be related to #7809 . Granted I know nothing about code so they might be totally unrelated.

### PR DESCRIPTION
**Pull request by Kyep**
🆑 Kyep
fix: Blood decals are no longer movable. This means LINDA cannot move them, spiders cannot wrap them, etc.
/🆑

### CODE REVIEW

File: code/modules/mob/living/simple_animal/hostile/terror_spiders/actions.dm [hunk 1]
```diff
@@ -196,7 +196,7 @@
 					choices += L
 		for(var/obj/O in oview(1,src))
 			if(Adjacent(O) && !O.anchored)
-				if(!istype(O, /obj/structure/spider/terrorweb) && !istype(O, /obj/structure/spider/cocoon) && !istype(O, /obj/structure/spider/spiderling/terror_spiderling))
+				if(!istype(O, /obj/structure/spider/terrorweb) && !istype(O, /obj/structure/spider/cocoon) && !istype(O, /obj/structure/spider/spiderling/terror_spiderling) && !istype(O, /obj/effect))
```
Comment (Crazylemon64) [inline code/modules/mob/living/simple_animal/hostile/terror_spiders/actions.dm]: No. The issue is that bloodstains are not anchored.

Comment (taukausanake) [inline code/modules/mob/living/simple_animal/hostile/terror_spiders/actions.dm]: So is that also why atmos can move bloodstains? Weird, I thought decals were by default anchored. At least the mess decals

Comment (Kyep) [inline code/modules/mob/living/simple_animal/hostile/terror_spiders/actions.dm]: Are you sure, Crazy?

/obj/effect includes things like bluespace portals, beams, forcefields, immovable rods, meteors, blood runes, sparks, steam, smoke, etc.

I don't see why any of these things should be wrappable.

**Comment by Kyep** [comment]
@Crazylemon64 Changed

File: code/game/objects/effects/decals/cleanable.dm [hunk 2]
```diff
@@ -2,6 +2,7 @@
 	var/list/random_icon_states = list()
 	var/noscoop = 0   //if it has this, don't let it be scooped up
 	var/noclear = 0    //if it has this, don't delete it when its' scooped up
+	anchored = 1
```
Comment (Fox-McCloud) [inline code/game/objects/effects/decals/cleanable.dm]: pre-defind vars before new ones, please.

Comment (Kyep) [inline code/game/objects/effects/decals/cleanable.dm]: Changed

**Comment by Kyep** [comment]
@Fox-McCloud Changed

**Pull request merged**